pytest_splinter/plugin.py

```diff
@@ -305,6 +305,12 @@ def get_args(driver=None,
         for key, value in firefox_profile_preferences.items():
             profile.set_preference(key, value)
         kwargs['firefox_profile'] = profile.encoded
+
+        # remote geckodriver does not support the firefox_profile desired
+        # capatibility. Instead `moz:firefoxOptions` should be used:
+        # https://github.com/mozilla/geckodriver#firefox-capabilities
+        kwargs['moz:firefoxOptions'] = {}
```

Comment:
Good point. I will change it to take care of not overwriting any existing `moz:firefoxOptions` / `profile`!